src/lib/graphqlErrorHandler.ts

```diff
@@ -108,11 +109,18 @@ export const formattedGraphQLError = (
   }
 
   const httpStatusCodes: number[] = []
-  ;(flattenedErrors || flattenErrors(topLevelError)).forEach(
-    e =>
+  ;(flattenedErrors || flattenErrors(topLevelError)).forEach(e => {
+    // Check for server-side errors during stitching downstream
+    // TODO: Get typing working for `ServerError`
+    // https://github.com/apollographql/apollo-link/blob/480df382cf7db486ae76c56ac2522134d77e36fa/packages/apollo-link-http-common/src/index.ts#L15
```

Comment:
You seem to have been able to get this type in the tests, right? So can you elaborate on this TODO?

Reply:
Can you show what that code looked like?

Reply:
Our `HTTPError` class is a class and thus has instances, additionally it’s a runtime class that you can check with runtime code. `ServerError` seems to be an interface for plain objects, and interfaces exist only in a type system so no runtime assertions can be made against them.

What you can do is write a TS ‘type guard’: https://basarat.gitbooks.io/typescript/docs/types/typeGuard.html

Reply:
I kind of like the `get` helper here anyway, so I've updated to remove the TODO and keep it as-is. If we have to fine-tune it more and refactor we can cast/guard things properly.

Reply:
I think I understand what you're saying, `ServerError` is just a type and so can't really be used in your actual runtime code (it doesn't exist in JS). It can only be used...to type things!